CombineAIDriver.lua

```diff
@@ -79,6 +86,7 @@ function CombineAIDriver:init(vehicle)
 		local implementWithPipe = AIDriverUtil.getAIImplementWithSpecialization(self.vehicle, Pipe)
 		if implementWithPipe then
 			self.pipe = implementWithPipe.spec_pipe
+            self.combine.spec_pipe = self.pipe
```

Comment:
Is this really necessary? Why?

Reply:
No, it's not. For some reason, I thought the Premos didn't have a spec_pipe but never really tested it.